content/en/docs/setup/independent/high-availability.md

```diff
@@ -10,513 +10,217 @@ content_template: templates/task
-    * `priority` should be higher for the first master node, e.g. 101, and lower for the others, e.g. 100.
-    * `auth_pass` use any random string here.
-    * `virtual_ipaddresses` should contain the virtual IP for the master nodes.
-
-3. Install the following health check script to _/etc/keepalived/check_apiserver.sh_ on all master nodes:
-     ```bash
-     #!/bin/sh
-
-     errorExit() {
-        echo "*** $*" 1>&2
-        exit 1
-     }
+At the end of this step, try `ssh`ing between nodes to ensure success.
 
-     curl --silent --max-time 2 --insecure https://localhost:6443/ -o /dev/null || errorExit "Error GET https://localhost:6443/"
-     if ip addr | grep -q <VIRTUAL-IP>; then
-        curl --silent --max-time 2 --insecure https://<VIRTUAL-IP>:6443/ -o /dev/null || errorExit "Error GET https://<VIRTUAL-IP>:6443/"
-     fi
-     ```
-
-    Replace the `<VIRTUAL-IP>` by your chosen virtual IP.
+{{% capture steps %}}
 
-4. Restart keepalived. While no Kubernetes services are up yet it will log health check fails on all master nodes. This will stop as soon as the first master node has been bootstrapped.
-{{% /tab %}}
-{{< /tabs >}}
+# External Etcd -- The instructions
 
+This guide assumes calico will be used as the pod network and will therefore use
+some default values, but they will be called out explicitly so replacing calico
+with another pod network is possible.
 
-## Acquire etcd certs
+## Setting up an HA etcd cluster using kubeadm
 
-Only follow this step if your etcd is hosted on dedicated nodes (**Option 1**). If you are hosting etcd on the masters (**Option 2**), you can skip this step since you've already generated the etcd certificates on the masters.
+Please see [setting up an HA etcd cluster using kubeadm](/docs/tasks/administer-cluster/setup-etcd-with-kubeadm).
 
-1. Generate SSH keys for each of the master nodes by following the steps in the [create ssh access](#create-ssh-access) section. After doing this, each master will have an SSH key in `~/.ssh/id_rsa.pub` and an entry in `etcd0`'s `~/.ssh/authorized_keys` file.
+After this setup you should have these files on the first bootstrapped member.
+You will need to put these files on all of your control-plane nodes.
 
-1. Run the following:
-     ```bash
-     mkdir -p /etc/kubernetes/pki/etcd
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/ca.pem /etc/kubernetes/pki/etcd
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client.pem /etc/kubernetes/pki/etcd
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client-key.pem /etc/kubernetes/pki/etcd
-     ```
+1. `/etc/kubernetes/pki/etcd/ca.crt`
+1. `/etc/kubernetes/pki/apiserver-etcd-client.crt`
+1. `/etc/kubernetes/pki/apiserver-etcd-client.key`
 
-## Run `kubeadm init` on `master0` {#kubeadm-init-master0}
+Here is one way to do it, but there are many reasons why this might not work for
+you. For instance, you may be running any other linux distro.
```

Comment:
(continue as block within the numbered list -- lmk if you need help figuring out the markup cuz it's different in Hugo/Blackfriday, although I think there might be OK models in some of the content you deleted). Also edit:

For example, you can run the following:

(note note that I suggest to follow)